### PR TITLE
Feature show coverage

### PR DESCRIPTION
**Pull request by danielbrodin**
Hi

This PR will add a status bar integration that shows the current files flow coverage.
I hope you don't mind, but I added a `.editorconfig` to fix some code standards for me.

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -121,4 +123,36 @@ export default {
     }
     return provider
   },
+
+  consumeStatusBar(statusBar: any): void {
+    this.coverageView = new CoverageView()
+    this.coverageView.initialize()
+    this.statusBar = statusBar.addLeftTile({ item: this.coverageView, priority: 10 })
+
+    this.subscriptions.add(atom.workspace.onDidChangeActivePaneItem((item: TextEditor|null|any): void => {
```
Comment (steelbrain) [inline lib/index.js]: As a type for item, use `?TextEditor`

File: lib/index.js [hunk 2]
```diff
@@ -121,4 +123,36 @@ export default {
...
+      if (item && item instanceof TextEditor) {
+        this.updateCoverage(item)
+      }
```
Comment (steelbrain) [inline lib/index.js]: add an else case to set coverage to zero when the editor is not text editor

File: lib/index.js [hunk 3]
```diff
@@ -121,4 +123,36 @@ export default {
...
+    const executable: string = await this.getExecutablePath(fileDirectory)
+    try {
+      const result: string = await exec(executable, ['coverage', filePath, '--json'], { cwd: fileDirectory, ignoreExitCode: true })
+
+      if (result) {
+        const coverage: CoverageObject = JSON.parse(result)
+        this.coverageView.update(coverage)
+      }
+    } catch (error) {
+      // Let the linter handle any flow errors.
```
Comment (steelbrain) [inline lib/index.js]: Again, handle case and update coverage to unknown

File: lib/index.js (hunk 1 above)
Comment (danielbrodin) [inline lib/index.js]: Nice, knew there had to be a better way to do it. Completely new to flow :)

**Comment by steelbrain** [comment]
Thank You again for working on this 👍

**Comment by steelbrain** [comment]
Published in v1.2 🎉

**Pull request merged**